src/Nut/Extensions.php

```diff
@@ -25,20 +25,20 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        if (count($this->app['extend.manager']->messages)) {
-            foreach ($this->app['extend.manager']->messages as $message) {
+        if (count($this->app['extend.manager']->getMessages())) {
+            foreach ($this->app['extend.manager']->getMessages() as $message) {
                 $output->writeln(sprintf('<error>%s</error>', $message));
             }
             return;
         }
 
-        $installed = $this->app['extend.manager']->showPackage('installed');
+        $allPackages = $this->app['extend.manager']->getAllPackages();
         $rows = [];
 
-        foreach ($installed as $ext) {
-            /** @var \Composer\Package\CompletePackageInterface $package */
-            $package = $ext['package'];
-            $rows[] = [$package->getPrettyName(), $package->getPrettyVersion(), $package->getType(), $package->getDescription()];
+        foreach ($allPackages as $type => $packages) {
+            foreach ($packages as $package) {
+                $rows[] = [$package['name'], $package['version'], $package['type'], $package['descrip']];
+            }
         }
 
         $table = $this->getHelper('table');
```

Comment:
This part isn't correct, and also breaks the command. Drop that part and we have merge material :grin: